core-api/src/main/java/com/sequenceiq/cloudbreak/client/RestClientUtil.java

```diff
@@ -55,33 +42,21 @@ public static synchronized Client get(ConfigKey configKey) {
 
     public static Client createClient(String serverCert, String clientCert, String clientKey, boolean debug, Class debugClass) throws Exception {
         SSLContext sslContext = SSLContexts.custom()
-                .loadTrustMaterial(KeyStoreUtil.createTrustStore(serverCert), null)
-                .loadKeyMaterial(KeyStoreUtil.createKeyStore(clientCert, clientKey), "consul".toCharArray())
-                .build();
+            .loadTrustMaterial(KeyStoreUtil.createTrustStore(serverCert), null)
+            .loadKeyMaterial(KeyStoreUtil.createKeyStore(clientCert, clientKey), "consul".toCharArray())
+            .build();
 
         ClientConfig config = new ClientConfig();
         config.property(ClientProperties.FOLLOW_REDIRECTS, "false");
-
-        RegistryBuilder<ConnectionSocketFactory> registryBuilder = RegistryBuilder.create();
-        registryBuilder.register("http", PlainConnectionSocketFactory.getSocketFactory());
-        registryBuilder.register("https", new SSLConnectionSocketFactory(sslContext));
-
-        PoolingHttpClientConnectionManager connectionManager =
-                new PoolingHttpClientConnectionManager(registryBuilder.build());
-
-        connectionManager.setMaxTotal(MAX_TOTAL_CONNECTION);
-        connectionManager.setDefaultMaxPerRoute(MAX_PER_ROUTE_CONNECTION);
-
-        // tell the jersey config about the connection manager
-        config.property(ApacheClientProperties.CONNECTION_MANAGER, connectionManager);
-        config.connectorProvider(new ApacheConnectorProvider());
+        config.property(ClientProperties.CONNECT_TIMEOUT, CONNECT_TIMEOUT_MS);
```

Comment:
Do we need to change the connect timout? What is the problem with the standard value?

Reply:
It is set to infinite